modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/VirtualFlowTest.java

```diff
@@ -34,6 +34,7 @@
 import static org.junit.Assert.assertSame;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
```

Comment:
We do not want to mix JUnit 4 and JUnit 5 calls in the same test class. Since converting this class to use JUnit 5 would be out of scope for a bug fix such as this, please rewrite it using JUnit 4 API calls only (which probably means a try/catch with a "fail" if you don't get the expected exception.